src/reference/torii-endpoints.md

```diff
@@ -217,11 +270,105 @@ A `GET` request to the endpoint.
 "Healthy"
 ```
 
-## Metrics
+## Query
+
+- **Protocol**: `HTTP`
+- **Method**: `POST`
+- **Encoding**: `SCALE`
+- **Endpoint**: `/query`
+
+#### Requests
+
+This endpoint expects requests with two shapes:
+
+Start a query:
+
+- **Body**: [`SignedQuery`](/reference/data-model-schema#signedquery)
+
+OR
+
+Get the next batch of a previously started query:
+
+- **Parameters**:
+  - `cursor` - specifies a cursor previously returned as part of query response
+
+Request
```

Comment:
It is unclear what "Request" specifically does here. Is it part of the sentence in the previous line, is it leftover of something?

Reply:
Oops, this seems to be unfinished. I will rewrite this piece.

Reply:
Simplified this section. And removed some other verbose `#### Requests` sections.